ansible/install.yml

```diff
@@ -30,7 +30,7 @@
      apt: update_cache=yes
      
    - name: install python-dev
-     apt: pkg=python-dev state=installed
+     apt: pkg=python3-dev state=installed
```

Comment:
Not sure about this. Shouldn't this be in an `install_py3.yml`and not in the default installation?